src/v0/destinations/ga4/config.js

```diff
@@ -60,22 +60,26 @@ const ConfigCategory = {
   PRODUCT_CLICKED: {
     name: 'GA4ProductClickedConfig',
     event: 'select_item',
+    itemList: 'NO',
```

Comment:
`ItemList` means we are expecting `Products` Array in input `No` means `optional`

Reply:
`ItemList` change for this particular event is reverted. Because for product clicked event, we can map root-level properties to GA4 `items` array and we don't require any backward compatibility of `products` array mapping here.